content/en/docs/reference/glossary/control-plane.md

```diff
@@ -2,6 +2,6 @@
 title: Control Plane
 ---
 
-A control plane is a set of one or more instances that share the
-same configuration source. The control plane configures the mesh or a subset of
-the mesh to manage the communication between the workload instances within.
+A control plane configures the mesh or a subset of the mesh to manage
```

Comment:
This change tells readers what a control plane does but not what it is. Keep in mind that definitions must follow a specific logical structure:
1. The term (word or phrase) to be defined
2. The class of object or concept to which the term belongs
3. The differentiating characteristics that distinguish it from all others of its class
Source: https://owl.purdue.edu/owl/general_writing/common_writing_assignments/definitions.html

Note that the original text follows that pattern whereas your proposed change doesn't.

Reply:
All I did was change the order of the original two sentences, because the first sentence did not say what it is (its purpose), but rather just one of it's characteristics. It just said `one or more instances ...`. What's an instance, and what does it do? The second sentence explained what its for.

How about this:
```suggestion
A control plane is a service that configures the mesh or a subset of the mesh to manage
```

Reply:
I would strongly disagree with that statement because it's my understanding that the control plane is a `set of processes/instances` and that text would then equate to stating that a service is a `set of processes/instances`, which is highly contentious, AFAIK. @smawson WDYT?

Reply:
I'm very confused. I'm using the term service generically (no glossary reference to Istio Service). It does provide a service with no further implications.

That said, a control plane actually is `a set of services` (processes/instances are the instances).

Anyway, to avoid ambiguity, how about: `is a component that ...`.

Otherwise, what do you suggest? What is a control plane and what's it for?

Wasn't this one of Costin's requested changes too?

Reply:
Another suggestion is `a set of control processes ...` or `a control service ...`.

Reply:
After thinking about it more, `a set of system services` would seem to be most correct.